MultiActionButton TypeScript children property failing

### Current behaviour

When making a push form the local repo which uses the component i cannot do a commit as the Husky tool receives an error coming from the MultiActionButton,

```
 error TS2322: Type '{ children: Element[]; text: string; buttonType: "secondary"; size: string; }' is not assignable to type 'IntrinsicAttributes & MultiActionButtonProps'.
  Property 'children' does not exist on type 'IntrinsicAttributes & MultiActionButtonProps'.

 <MultiActionButton
```


### Expected behaviour

be able to use the MultiActionButton without problems

### CodeSandbox or Storybook URL

https://carbon.sage.com/storybook?path=/docs/multi-action-button--default-story

### JIRA Ticket (Sage Only)

_No response_

### Suggested Solution

_No response_

### Carbon Version

https://github.com/Sage/carbon/releases/tag/v106.0.0

### What browsers are you seeing the problem on?

Other

### What Operating System are you seeing the problem on?

MacOS

### Anything else we should know?

_No response_

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

--- a/src/components/split-button/index.d.ts
+++ b/src/components/split-button/index.d.ts
@@ -1,1 +1,2 @@
 export { default } from "./split-button";
+export type { SplitButtonProps } from "./split-button";

